requirements/common.txt

```diff
@@ -1,7 +1,7 @@
 cachetools
 psutil
 sentencepiece  # Required for LLaMA tokenizer.
-numpy < 2.0.0
+numpy
```

Comment:
Actually do we need to restrict numpy<2 for python_version == '3.9' since we are still using numba 0.60?

Reply:
Fixed.

Reply:
This doesn't look fixed?